frontend/integration-tests/tests/login.scenario.ts

```diff
@@ -1,4 +1,9 @@
 import { $, browser, ExpectedConditions as until } from 'protractor';
+import {
+  sideHeader,
+  switchPerspective,
+  Perspective,
+} from '@console/dev-console/integration-tests/views/dev-perspective.view';
```

Comment:
I'm debating if these should move since they're no longer specific to the dev console tests, maybe frontend/integration-tests/views/sidenav.view.ts

@christianvogt opinion?

Reply:
My first instinct was to move them as you suggested but then didn't do it. I also think it would be better to just move them here.

Reply:
@spadgett I moved the necessary code in `sidenav.view.ts`.

Reply:
We can remove it from `dev-console/integration-tests` later as none of the dev console based scenarios are being run along with other e2e tests right now. They are outdated and need to be updated based on my last conversation with our QE team.